src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,32 @@
+package seedu.addressbook.data.tag;
+import seedu.addressbook.data.person.Person;
+
+/**
+ * Represents an addition or deletion of a tag to a Person
+ */
+
+public class Tagging {
+
+    Person person;
+    Tag tag;
+    boolean added;
+    
+    Tagging(Person person, Tag tag, boolean added) {
+        this.person = person;
+        this.tag = tag;
+        this.added = added;
+    }
+    
+    public String getPrintableTagString() {
```

Comment:
You may want to implement the `Printable` interface in your class definition since you write this method.